File: Python/0260_single_number_III/solution.py
def find_single_number(nums):
    refer_dict = dict()
    for num in nums:
        if num not in refer_dict:
            refer_dict[num] = 1
        elif num in refer_dict:
            refer_dict[num] += 1
    unique = list()
    for key,value in refer_dict.items():
        if value == 1:
            unique.append(key)
    return unique

nums = [1,2,1,3,2,5]
print(find_single_number(nums))

File: JavaScript/0260_single_number_III/solution.js
function find_single_number(nums) {
    let refer_dict = new Object();
    for (let num of nums) {
        if (!(num in refer_dict)) {
            refer_dict[num] = 1
        } else if (num in refer_dict) {
            refer_dict[num] ++;
        }
    }
    let unique = new Array();
    for (let [key,value] of Object.entries(refer_dict)) {
        if (value == 1) {
            unique.push(key);
        }
    }
    return unique;
}

const nums = [1,2,1,3,2,5];
console.log(find_single_number(nums));